modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,271 @@
++
+.Example usage
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: pipeline-param-enum
+spec:
+  params:
+  - name: message
+    enum: ["v1", "v2"]
+    default: "v1"
+# ...
+----
+
+* With this update, when using the Git resolver with the authenticated SCM API, you can override the default token, SCM type, and server URL that you configured.
++
+.Example usage
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: git-api-demo-tr
+spec:
+  taskRef:
+    resolver: git
+    params:
+    - name: org
+      value: tektoncd
+    - name: repo
+      value: catalog
+    - name: revision
+      value: main
+    - name: pathInRepo
+      value: task/git-clone/0.6/git-clone.yaml
+    # create the my-secret-token secret in the namespace where the
+    # pipelinerun is created. The secret must  contain a GitHub personal access
+    # token in the token key of the secret.
+    - name: token
+      value: my-secret-token
+    - name: tokenKey
+      value: token
+    - name: scmType
+      value: github
+    - name: serverURL
+      value: https://ghe.mycompany.com
+----
+
+* With this update, you can define default resource requirements for containers and init-containers in poids that {pipelines-shortname} creates when executing task runs. Use the `pipeline.options.configMaps.config-defaults.default-container-resource-requirements` spec in the `TektonConfig` CR to set these requirements. You can set the default values for all containers and also for particular containers by name or by prefix, such as `sidecar-*`.
+
+[id="operator-new-features-1-14_{context}"]
+=== Operator
+
+* With this update, {pipelines-shortname} supports horizontal pod autoscaling for the operator proxy webhook. If the pod that runs the operator proxy webhook reaches 85% CPU utilization, the autoscaler creates another replica of the pod. If you want to use more than one replica for the operator proxy webhook at startup, you must configure this number in the `options.horizontalPodAutoscalers` spec of the `TektonConfig` CR.
+
+* With this update, the internal leader election for several components of {pipelines-shortname} was improved. The operator controller, operator webhook, proxy webhook, {pac} watcher, {pac} webhook, and the Tekton Chains controller now use separate leader election ConfigMaps. The leader election affects which replica of a component processes a request.
+
+* Before this update, when you scaled up the number of replicas of the {pipelines-shortname} controller, manual intervention was required to enable the use of the new replicas, namely, you needed to delete leases in the leader election. With this update, when you scale up the number of replicas of the {pipelines-shortname} controller, the leader election includes the new replicas automatically, so the new replicas can process information.
+
+* With this update, you can optionally set the following flags in the pipeline spec:
```

Comment:
@piyush-garg shouldn't this inside `tektonconfig` under `spec.pipeline` section?